contrastRatio.js

```diff
@@ -38,6 +38,7 @@ const ContrastRatio = new Lang.Class({
 
         this._ratioLabel = new St.Label()
         this._scoreLabel = new St.Label()
+        this._scoreLabel.set_style('font-weight: bold;');
```

Comment:
Would you mind using a style class instead?

Reply:
Hey, I don't know how to use style class (barely know how to use gnome, luckly your code is quite self-explanatory), and I couldn't find much documentation, do you mind providing a link to the style class? 
How should it look?
something like this?
```js
const scoreStyle = SomeStyle({"font-weight": "bold"});
this._scoreLabel = new St.Label(scoreStyle)
```

Reply:
I believe something like this would work:

```javascript
this._scoreLabel = new St.Label({ reactive: false, style_class: 'your-class-name' })
```

I'm not on linux right now so I can't double check, but let me know if that gives you the results you expect.